npm-scripts.js

```diff
@@ -258,15 +277,18 @@ function cleanWorker()
 
 	if (isWindows)
 	{
-		executeCmd('rd /s /q worker\\out\\msys');
+		if (fs.existsSync('worker/out/msys'))
+		{
+			executeCmd('rd /s /q worker\\out\\msys');
+		}
 	}
 }
 
 function lintNode()
 {
 	console.log('npm-scripts.js [INFO] lintNode()');
 
-	executeCmd('eslint -c node/.eslintrc.js --max-warnings 0 node/src node/.eslintrc.js npm-scripts.js worker/scripts/gulpfile.js');
+	executeCmd('eslint --ignore-path node/.eslintignore -c node/.eslintrc.js --max-warnings 0 node/src node/.eslintrc.js npm-scripts.js node/src/tests worker/scripts/gulpfile.js');
```

Comment:
Why have you added `node/src/tests` here? `node/src` is already present.

Reply:
true